src/platform/backend/drm.c

```diff
@@ -141,19 +144,75 @@ create_fb(struct gbm_surface *surface, struct drm_fb *fb)
    return false;
 }
 
+static uint32_t
+create_dumb_fb(uint32_t width, uint32_t height)
+{
+   struct drm_mode_destroy_dumb destroy_request = { 0 };
+   struct drm_mode_map_dumb map_request = { 0 };
+   uint8_t *map;
+   uint32_t fd;
+
+   struct drm_mode_create_dumb create_request = { 0 };
+   create_request.width = width;
+   create_request.height = height;
+   create_request.bpp = 32; /* RGBX8888 */
+
+   int ret;
+   ret = drmIoctl(drm.fd, DRM_IOCTL_MODE_CREATE_DUMB, &create_request);
```

Comment:
You can just directly check the return values here. Temporary ret serves no purpose.